packages/cli/src/cli.ts

```diff
@@ -23,6 +23,10 @@ export const cmd = yargs(hideBin(process.argv))
     description: 'Set the default log level to none, default, info or debug',
     array: true,
   })
+  .option('log-json', {
+    description: 'Output all logs as JSON objects',
+    boolean: true,
+  })
```

Comment:
We need to make a concerted effort to reflect the flag available on the CLI at least on the README for the cli package.

Reply:
Happy to be noisier about it but I think json logging is more of a system admin thing and less a general user thing?

Reply:
Yeah thats cool, when I looked that the README for the CLI I realised we need much more than just documenting the json logging thing.. so lets leave that for now and come back to do a big whack of docs on the CLI for newcomers at a later point.